commands/fork.go

```diff
@@ -73,9 +92,9 @@ func fork(cmd *Command, args *Args) {
 		}
 	} else {
 		if !args.Noop {
-			newRepo, err := client.ForkRepository(project)
+			newRepo, err := client.ForkRepository(project, params)
 			utils.Check(err)
-			forkProject.Owner = newRepo.Owner.Login
```

Comment:
This line should stay the way it was. We want to read the information about the new repo from the response from the server.